src/app/auth/sign-up/sign-up.component.html

```diff
@@ -0,0 +1,124 @@
+<div class="row progress-row">
+  <div class="col-4 sections-col" *ngFor="let sectionIcon of sectionsIcons; let i = index">
+    <div class="row">
+      <div class="col-6">
+        <object
+          [ngClass]="{'highlighted': i + 1 <= section}"
+          type="image/svg+xml"
+          [data]="sectionIcon | sanitizeUrl"
+          class="sections-icon">
+          Your browser does not support SVG
+        </object>
+      </div>
+      <div class="col-6" *ngIf="i != sectionsIcons.length - 1">
+        <hr [ngClass]="{'highlighted': i + 1 < section}">
+      </div>
+    </div>
+  </div>
+</div>
+
+<div id="signup-section">
+
+  <!--    <app-alert-->
+  <!--      [message]="error"-->
+  <!--      *ngIf="error"-->
+  <!--      (close)="onHandleError()"></app-alert>-->
+
+  <!--    <div *ngIf="isLoading" align="center">-->
+  <!--      <app-loading-spinner></app-loading-spinner>-->
+  <!--    </div>-->
+
+  <form [formGroup]="signUpForm1"
```

Comment:
It would be better to create a separate component for every form. It will improve the readability and testability main component. Don't forget  about ChangeDetectionStrategy

Reply:
Created 3 separate components and moved logic and data